LLMonFHIR.xcodeproj/project.pbxproj

```diff
@@ -100,6 +102,8 @@
 		2FE5DC4229EDD7F2004B9AB4 /* Binding+Negate.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "Binding+Negate.swift"; sourceTree = "<group>"; };
 		2FE5DC4329EDD7F2004B9AB4 /* Bundle+Image.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "Bundle+Image.swift"; sourceTree = "<group>"; };
 		2FE5DC4429EDD7F2004B9AB4 /* CodableArray+RawRepresentable.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = "CodableArray+RawRepresentable.swift"; sourceTree = "<group>"; };
+		4333523F2A5C95E00043A440 /* MultipleResourceChat.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = MultipleResourceChat.swift; path = "../../../../Downloads/LLMonFHIR-multiple_resource_interpreter 2/LLMonFHIR/FHIR Display/MultipleResourceChat.swift"; sourceTree = "<group>"; };
+		433352432A5C96090043A440 /* FHIRMultipleResourceInterpreter.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = FHIRMultipleResourceInterpreter.swift; path = "../../../../Downloads/LLMonFHIR-multiple_resource_interpreter 2/LLMonFHIR/FHIR Display/FHIRMultipleResourceInterpreter.swift"; sourceTree = "<group>"; };
```

Comment:
Seems like the files were added with non-relative to the surrounding folder setting. You can change this in the Xcode "File Inspector" panel on the trailing side of the Xcode window. I think that "Relative to Group" should be the correct setting that we should aim for.